jl4-core/src/L4/Lexer.hs

```diff
@@ -42,8 +42,6 @@ data PosToken =
   deriving stock (Eq, Ord, Show, Generic)
   deriving anyclass (ToExpr, NFData)
 
-instance ToExpr NormalizedUri
```

Comment:
Why?

Reply:
Move it further down the dependency tree. We dont particularly need it (again, because I reverted some changes) but it also can’t hurt to have it as far down as possible for the future.